ang/civicase/activity/feed/directives/activity-feed.directive.js

```diff
@@ -479,6 +472,26 @@
       });
     }
 
+    /**
+     * Prepares Activity Params
```

Comment:
Why do we want to explicitly say that `Prepares Activity Params` when the name `prepareActivityParams` itself says that. 
Can't we just change it to -
```
    /**
     * Adds Action Links chained Api call
     *
     * @param {Object} returnParams
     * @param {Object} params
     *
     * @return {Array}
     */
```